### ISSUE TITLE
Пару вопросов от новичка

### ISSUE THREAD

**Issue by 6pm** [57uff3r/nodejs-vksdk#37]
Я хочу сделать приложение на react native с использованием данных из vk(музыка в основном). И у меня есть пару вопросов:
1. Создал приложение, подключил - в настройках прописал сервер localhost. Далее делаю запрос к api и получаю ошибку в консоли:
`XMLHttpRequest cannot load https://api.vk.com/method/users.get. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:8081' is therefore not allowed access.` Проблема в том, что надо запускать с домена?
2. Если запускать с домена, то подойдет ли простой хостинг без node.js?
3. Чем отличается ваш модуль от оф. vk javascript api?
Cпасибо за ответ.

**Comment by 57uff3r** [comment]
Привет!

Эта библиотека предназначена для запуска исключительно на node сервере, в браузерных или react-native приложениях эта штука работать не будет.  
В этом как раз и есть отличие — библиотека предназначена на 100% для серверной разработки, в то время как официальные клиенты для  VK API — для клиентской.

**Comment by 6pm** [comment]
Получается что возможности те же, только разница между ними в среде запуска?

**Comment by 57uff3r** [comment]
Да, основное различие в этом. + есть методы vk api, которые работают только с бэкенда и не рассчитаны на запуск в браузере/в приложении (проверка баланса, отправка уведомлений и т.д.)

**Comment by 6pm** [comment]
У вас есть какое либо приложение или сайт использующий ваш пакет? Если можно, скиньте линк.

**Comment by digitalhitler** [comment]
Почитайте про CORS, там все просто на самом деле. Кстати, странно, что при исполнении кода на сервере он у вас ругается на глубоко браузерные штуки.
